src/_includes/partials/author_bios/sanjeetsuhag.md

```diff
@@ -0,0 +1 @@
+3D Software Developer at Cesium
```

Comment:
Can you fill this out so it has a few sentences? It should start with “Sam Suhag is a….” You can look at other files in this directory as examples.